.circleci/config.yml

```diff
@@ -0,0 +1,105 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
+# DOCKER_USER
+# DOCKER_PASS
+# DOCKER_DEPLOY - set to true to enable pushing to Dockerhub
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # base image
+      - image: ubuntu:16.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install essential packages
+          command: |
+            apt-get update
+            apt-get install -y ca-certificates curl build-essential make git
+            curl -L -o /tmp/dockerize.tgz https://github.com/jwilder/dockerize/releases/download/v0.6.1/dockerize-linux-amd64-v0.6.1.tar.gz
+            tar -xz -C /usr/bin -f /tmp/dockerize.tgz
```

Comment:
Do I understand correctly, that this is only necessary for running the (separate) database image used for testing the app, and doesn't actually go into the production build?

Reply:
Correct. I've added a comment to clarify this.